docs/diagnostics/PX1054.md

```diff
@@ -0,0 +1,30 @@
+# PX1054
+This document describes the PX1054 diagnostic.
+
+## Summary
+
+| Code   | Short Description                                                                                 | Type  | Code Fix    | 
+| ------ | ------------------------------------------------------------------------------------------------- | ----- | ----------- | 
+| PX1054 | A `PXGraph` instance cannot start a long-running operation during the `PXGraph` initialization. | Error | Unavailable |
+
+## Diagnostic Description
+A `PXGraph` instance cannot start a long-running operation in `PXGraph` constructors or in the `Initialize` method overridden in a graph extension.
+
+To fix the issue, you remove the invocation of a long-running operation from the `PXGraph` constructor or the `Initialize` method overridden in a graph extension and rework the related business logic.
+
+## Example of Incorrect Code
+
+```C#
+public class DocSOInvoiceProcess : PXGraph<DocSOInvoiceProcess>
+{
+    public DocSOInvoiceProcess()
+    {
+        Invoices.SetProcessDelegate(delegate (List<DocInvoiceLog> list) // The PX1054 error is displayed for this line.
```

Comment:
@dropsonic , changed the example, but I'm not sure for which line the error is displayed and I cannot verify it because my version of Acuminator does not include this diagnostic.